dist/index.html

```diff
@@ -4,9 +4,57 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="index.css">
   <title>Document</title>
 <script defer src="main.js"></script></head>
 <body>
+  <header>
+    <nav class="container">
+      <h1 class="header">Awesome Books</h1>
+      <ul class="nav">
+        <li class="menu" id="list">List</li>
+       
+        <li class="menu" id="add">Add</li>
+     
+        <li class="menu" id="contact">Contact</li>
+      </ul>
+    </nav>
+  </header>
+
+  <div class="time">
+    <p class="date" id="time"></p>
+  </div>
   
+  <main>
+    <section class="booklist">
+    
+    </section>
```

Comment:
- [ ] Please ensure that the booklist is styled as was provided in the wireframe.

| Your Implementation | The Wireframe |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/45002654/235493971-a9e3be40-8a0a-4284-8bcd-7351555255dd.png) | ![image](https://user-images.githubusercontent.com/45002654/235494033-7ceb4f83-bf12-449d-9e6d-081761a78650.png) |